docs/articles/getting-started.md

```diff
@@ -44,9 +43,10 @@ private async void button1_Click(object sender, EventArgs e)
 }
 ```
 
-If you then run your application and click the button you will see that all your chroma devices turned Red.
+If you then run your application and click the button you will see that all your chroma devices turned red.
+What `await` does is waiting for the initializiation of the Chroma SDK to finish before proceding. Methods using `await` not in the last line must be marked ss `async` (asynchronous) and thus allow the usage of `await` on the method itself. How this works under the hood, is that [Task](https://docs.microsoft.com/dotnet/csharp/programming-guide/concepts/async/) objects are returned instead of return values.
 
-### A note on [`ColoreProvider`](xref:Colore.ColoreProvider)
+#### A note on [`ColoreProvider`]((xref:Colore.ColoreProvider)
```

Comment:
The previous heading level was 2 (`##`), so this should stay as level 3 (`###`).

Also, the extra parentheses seems to have broken the link.

Reply:
I think it looks cleaner this way but at the end of the day it´s your docs.